lib/usdUfe/ufe/UsdUndoDuplicateSelectionCommand.cpp

```diff
@@ -236,4 +238,17 @@ void UsdUndoDuplicateSelectionCommand::undo() { _undoableItem.undo(); }
 
 void UsdUndoDuplicateSelectionCommand::redo() { _undoableItem.redo(); }
 
+Ufe::SceneItem::Ptr UsdUndoDuplicateSelectionCommand::targetItem(const Ufe::Path& sourcePath) const
+{
+    const auto it = std::find_if(
+        _duplicatedItemsMap.begin(), _duplicatedItemsMap.end(), [&sourcePath](const auto& pair) {
+            return pair.first->path() == sourcePath;
+        });
+    if (it == _duplicatedItemsMap.end()) {
+        return nullptr;
+    }
+
+    return Ufe::Hierarchy::createItem(it->second->path());
```

Comment:
Recreate the item in case it went stale.

I tried changing `_duplicatedItemsMap` to just store paths to prevent items from going stale in the first place, but that caused issues with the clipboard. If this command is used to duplicate `|stage1|stageShape1,/mtl/add1` to the clipboard, the UFE path of the duplicated item will just be `/add1` -- it's at the root of the clipboard stage.

`Ufe::Hierarchy::createItem()` is not able to create an item from `/add1`, so the clipboard relies on the actual item being stored and returned by `UsdUndoDuplicateSelectionCommand::getDuplicatedItemsMap()`.